pilot/pkg/features/pilot.go

```diff
@@ -704,6 +704,8 @@ var (
 
 	StackdriverAuditLog = env.Register("STACKDRIVER_AUDIT_LOG", false, ""+
 		"If enabled, StackDriver audit logging will be enabled.").Get()
+
```

Comment:
What are the considerations for enabling it if it is defaulted to false? I assume this is only as a short term measure till this is stabilized? If yes, may be we should default it to true. Also explain when this should be set to false

Reply:
I trade this as an experimental feature, so I think it should be off by default.
Like we did for native stats filter, it was first default off, then default on later.